cmake/Modules/LibAddPlugin.cmake

```diff
@@ -1,6 +1,41 @@
 include (LibAddMacros)
 include (LibAddTest)
 
+# ~~~
+# Provides additional compiler definitions:
```

Comment:
It would be good to have this docu also in the API docu of plugins or even in the plugin writing tutorial. But this can also be done in a later PR.